.buildkite/build_resources.py

```diff
@@ -27,6 +36,11 @@
         for v in json.loads(content):
             if v['filename'].startswith('resources/'):
                 pipeline['steps'][0]['skip'] = False
+                # We are doing this to override variable to point to the branch/PR
+                # from where the changes were pushed.
```

Comment:
We should also say that we cannot use the variable names directly because `yaml.dump` does not escape the quotes properly.